Campus Density.xcodeproj/project.pbxproj

```diff
@@ -159,6 +164,10 @@
 		B792838E50BEC810AD8F4BF1 /* Pods-Campus DensityTests.debug.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-Campus DensityTests.debug.xcconfig"; path = "Pods/Target Support Files/Pods-Campus DensityTests/Pods-Campus DensityTests.debug.xcconfig"; sourceTree = "<group>"; };
 		EE03ED8F23EBDA800064266B /* GoogleService-Info.plist */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.xml; name = "GoogleService-Info.plist"; path = "Secrets/GoogleService-Info.plist"; sourceTree = SOURCE_ROOT; };
 		EE03ED9023EBDA800064266B /* Keys.plist */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.xml; name = Keys.plist; path = Secrets/Keys.plist; sourceTree = SOURCE_ROOT; };
+		EE1BC55F24954238004DD427 /* AvailabilityInfoCell.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; name = AvailabilityInfoCell.swift; path = "/Users/anshgodha/Desktop/DTI/campus-density-ios/Campus Density/DiningCells/AvailabilityInfoCell.swift"; sourceTree = "<absolute>"; };
```

Comment:
This path should not be absolute btw, just change how the file is located
I found this because my build failed since my abs path is not the same (/Users/anshgodha/...)

Reply:
Okay! Thanks for that catch, I don't know why that specific path was like that. Will push out a commit